firefox-ios/Client/Experiments/Experiments.swift

```diff
@@ -127,6 +136,27 @@ enum Experiments {
 
         return url
     }()
+    
+    static let remoteSettingsService: RemoteSettingsService = {
```

Comment:
since retrieving `RemoteSettingsService` from profile isn't possible at the moment. We thought we could just create a separate one for Nimbus?

Reply:
ooh we can do that? ideally we we want to use the same `RemoteSettingsService`.
I may have completely misunderstood, when looking at it with @adudenamedruby, I think he said that there would be probably need changes like changing Experiments from an enum to a struct and some other steps and would be considered risky. 
I'm just a newbie so, im sure i don't have the context for everything 😅

Reply:
oooo. I totally forgot about the app container. I was looking at injecting the profile. But this works! THanks so much @issammani! Yeat, @tiftran, this is probably the way to go!!